charts/k8s-monitoring/templates/agent_config/_agent.river.txt

```diff
@@ -1,7 +1,11 @@
 {{ define "agent.config.agent.allowList" }}
-{{ if .Values.metrics.agent.metricsTuning.useDefaultAllowList }}
+{{ if or (index .Values.metrics "agent").metricsTuning.useDefaultAllowList (index .Values.metrics "agent").metricsTuning.useIntegrationAllowList }}
```

Comment:
```suggestion
{{ if or .Values.metrics.agent.metricsTuning.useDefaultAllowList .Values.metrics.agent.metricsTuning.useIntegrationAllowList }}
```

Reply:
We only need the `(index .Values.metrics "node-exporter")` because that's how you reference anything with a dash in the key. So, in this case, we can go back to the standard method.